exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/ParquetGroupScan.java

```diff
@@ -716,6 +726,7 @@ public String toString() {
     return "ParquetGroupScan [entries=" + entries
         + ", selectionRoot=" + selectionRoot
         + ", numFiles=" + getEntries().size()
+        + ", usedCache=" + usedMetadataCache
```

Comment:
This is a useful attribute; however usedCache might be misinterpreted to mean data cache. usedMetadataCache is more explicit (although the term 'cache' itself is misleading - here we are referring to file cache on disk, not in-memory). Another thing: is a boolean value sufficient ? if there are 2 subdirectories and only one of them has the metadata cache, and I query the parent directory: would the cache get used for one of the subdirectories and if so, what would this flag indicate ? It sounds like if the parent directory has the cache, then it will be true, otherwise false.